vnext/Microsoft.ReactNative/packages.config

```diff
@@ -2,6 +2,7 @@
 <packages>
   <package id="boost" version="1.76.0.0" targetFramework="native" />
   <package id="Microsoft.Build.Tasks.Git" version="1.0.0" targetFramework="native" developmentDependency="true" />
+  <package id="Microsoft.CodeAnalysis.BinSkim" version="1.9.0-prerelease1" targetFramework="native" />
```

Comment:
this nuget is 80 mb and contains a bunch of binaries for linux etc that we don't need. Using the GHA Task I mentioned above should get rid of this build time dependency :)

Reply:
Correct, And I beleive that all our customers would start downloading this package as well...